workflow/artifacts/git/git_test.go

```diff
@@ -25,6 +25,10 @@ func TestGitArtifactDriver_Load(t *testing.T) {
 		assert.DirExists(t, path)
 	})
 	t.Run("PrivateRepo", func(t *testing.T) {
+
+		// TODO: temp - skip private repo test for everyone
```

Comment:
why do you want to skip this test?

Reply:
As @alexec mentioned here
https://github.com/argoproj/argo-workflows/issues/8953#issuecomment-1282431447